src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -111,24 +115,34 @@ else if (SourceGroupPacketExtension.SEMANTICS_FID
                             grouped.add(rtxSSRC);
                         }
 
-                        int subjectiveQualityIdx = i * 3;
-                        RTPEncodingImpl encoding = new RTPEncodingImpl(
-                            track, subjectiveQualityIdx, primarySSRC, rtxSSRC,
-                            0, null);
-
-                        rtpEncodings[subjectiveQualityIdx] = encoding;
-
-                        subjectiveQualityIdx = i * 3 + 1;
-                        encoding = new RTPEncodingImpl(
-                            track, subjectiveQualityIdx, primarySSRC, rtxSSRC,
-                            1, new RTPEncodingImpl[] { encoding });
-                        rtpEncodings[subjectiveQualityIdx] = encoding;
+                        if (includeTemporal)
+                        {
+                            int subjectiveQualityIdx = i * 3;
+                            RTPEncodingImpl encoding = new RTPEncodingImpl(
+                                track, subjectiveQualityIdx, primarySSRC, rtxSSRC,
+                                0, null);
```

Comment:
could we describe the 0, 1, 2 used in the calls to `new RTPEncodingImpl` here?  even if it was with an inline comment like `..., 0 /* layer index */, ...`